### ISSUE TITLE
Post excerpt review

### ISSUE THREAD

**Issue by m-sullivan7** [cagov/cannabis.ca.gov#105]
From @chachasikes: 

> Extraction was a little ugly because their content intros have many patterns, but I pulled the first sentences and did a quick sanity pass on all of them.
> You can take a look in the "Posts" tab & edit any "excerpt" field content as it should appear on the landing pages when people paginate back into the past. Concentrating on the newest posts, like maybe the first 20-30 if you want a priority.
> "All Content" View

This ticket is to review the excerpts in Airtable and make any edits to provide useful excerpt descriptions to posts that will be featured on the _Announcements_ and _DCC events_ pages, especially to the most recent announcements, press releases, and events.

**Comment by m-sullivan7** [comment]
@chachasikes I reviewed all of the excerpts in the _Posts_ tab of Airtable since I'm waiting on more drafts. I made some nice-to-have updates and made a few definitely-change items. Is there anything else you need from me for the excerpts with the posts or am I OK to close this issue?

**Comment by m-sullivan7** [comment]
Confirmed with Chach that no further work is needed on this issue. Updated excerpts give her what she needs.
